src/modules/item/utils.ts

```diff
@@ -123,16 +118,17 @@ export function getBackgroundStyle(rarity?: ItemRarity) {
 // - Common: version:item_type:representation_id
 // - Wearables: version:item_type:representation_id:category:bodyshapes
 export function getMetadata(item: Item) {
-  const version = 1
-  const type = item.type[0]
-
   switch (item.type) {
     case ItemType.WEARABLE: {
       const data = item.data as WearableData
       const bodyShapeTypes = getBodyShapes(item)
         .map(toWearableBodyShapeType)
         .join(',')
-      return `${version}:${type}:${item.name}:${item.description}:${data.category}:${bodyShapeTypes}`
+
+      if (!data.category) {
```

Comment:
I'm honestly not sure when this would happen but I was following the `WearableData` type:
```ts
export type WearableData = {
  category?: WearableCategory
  (...)
```

(this didn't change I just added the check).
Previously we'd create an URN with an `:undefined:`, currently it'll blow up in the two sagas that use the method:
- Changing Item price and/or beneficiary 
- Publishing a Collection with the item
dispatching a failure